src/main/java/fi/metatavu/metaform/server/rest/RealmsApiImpl.java

```diff
@@ -59,16 +60,21 @@
   
   @Override
   public Response createReply(String realmId, UUID metaformId, Reply payload, Boolean updateExisting) throws Exception {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 21 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=13&issues=AWOR5dtrmXhJxGlVQKas&open=AWOR5dtrmXhJxGlVQKas)

[ISSUE_KEY]: AWOR5dtrmXhJxGlVQKas